backend/src/middleware/slugify/uniqueSlug.spec.js

```diff
@@ -19,3 +19,17 @@ describe('uniqueSlug', () => {
     expect(uniqueSlug(string, isUnique)).resolves.toEqual('anonymous')
   })
 })
+
+describe('Slug is transliterated correctly', () => {
+  it('Converts umlaut to a two letter equivalent', () => {
+    const umlaut = 'ä'
+    const isUnique = jest.fn().mockResolvedValue(true)
+    expect(uniqueSlug(umlaut, isUnique)).resolves.toEqual('ae')
```

Comment:
This is the message I get:
```
Expected: "ae"
Received: "a"
(node:18852) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:18852) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```
What we need is a `async` call:
```javascript
  it('Converts umlaut to a two letter equivalent', async () => {
  ...
  await expect(uniqueSlug(umlaut, isUnique)).resolves.toEqual('ae')
```
The you will get an error. It resolves to `a`